README.md

```diff
@@ -1,24 +1,32 @@
-# Frobnicator (this is a template)
+# RustyCards
 
 ## Authors
-- Andrzej Głuszak (@agluszak on GitHub)
-- Linus Torvalds (@torvalds on GitHub)
+- Igor Kamiński (@rog1gor on GitHub)
+- Jakub Owczarek (@obukaj on GitHub)
 
 ## Description
-Frobnicator is going to be a platformer game similar to Super Mario Bros made using Bevy game engine.
+Rusty Cards is going to be a duel, card game with online gameplay.
 
 ## Features
-- map generator
-- shooting
-- enemy AI
-- game state saving and loading
-- scores
+- Lobby that enables connection beetween two players with GUI
+- A duel with GUI:
+  - Drawing cards
+  - Gaining mana
+  - Droping cards into the battlefield
+  - Attacking with the cards on your battlefield
+
+## Later Features
+- Card collection
+- Customizable decks
 
 ## Plan
-In the first part we're going to implement the basics: movement, physics and shooting. The enemies will simply bounce from one edge of the platform to the other. There will be only a single map.
+In the first part we're going to implement the logic that stays behind the game. It will be possible  play the game with one deck, using command line (without GUI).
+
+In the second part we're going to create a lobby and add GUI.
 
-In the second part we're going to add random map generator, saving/loading, scores and a more sophisticated AI.
+Later we will try to develop the project - we will add more cards, card collection and we will enable deck custommization.
 
 ## Libraries
-- Bevy
 - Serde (for serialization)
```

Comment:
Nie jestem pewien, czy łączenie Bevy z Nannou to dobry pomysł: obydwa raczej działają z założeniem, że zarządzają własnym oknem i grafiką. Chyba lepiej byłoby do bevy użyć egui albo bevy_prototype_lyon (do rysowania prostych kształtów jak svg)